libraries/stdlib/src/kotlin/collections/Collections.kt

```diff
@@ -388,10 +389,12 @@ private fun rangeCheck(size: Int, fromIndex: Int, toIndex: Int) {
 
 @PublishedApi
 @SinceKotlin("1.3")
+@Suppress("NO_ACTUAL_FOR_EXPECT") // JS IR
```

Comment:
For JS_IR we copy JS sources during build, so this suppress should not be needed.